src/app/page/Request/foodbox/ConfirmStep.jsx

```diff
@@ -97,7 +95,6 @@ function ConfirmStep({ dispatch, state }) {
       const redirect = isDonationRequest ? "donation" : "payment";
       history.push(`/request/foodbox/success/${redirect}`);
     } catch (error) {
-      console.log(error);
```

Comment:
we need to discuss error handler, sentry perhaps ?

Reply:
Yes please